vcx/wrappers/java/src/main/java/com/evernym/sdk/vcx/credential/CredentialApi.java

```diff
@@ -28,6 +36,17 @@ public void callback(int command_handle, int err, int credentialHandle, String o
         }
     };
 
+
+    /**
+     * Create a credential based off of a known message id for a given connection.<br>
+     *
+     * @param sourceId Institution's personal identification for the credential, should be unique.
+     * @param connectionHandle connection to query for credential offer
+     * @param msgId msg_id that contains the credential offer
+     * @return A created credential
+     * @throws VcxException the vcx exception.
```

Comment:
I like the sentence "Thrown if an error occurs when calling the underlying SDK." rather than "the vcx exception". Can you fix this in all other places?

Reply:
You didn't fix this issue in the 'ConnectionApi.java' and the `CredentialApi.java`